doc/htconfig.md

```diff
@@ -48,12 +47,13 @@ line to your .htconfig.php:
 * ostatus_poll_timeframe - Defines how old an item can be to try to complete the conversation with it.
 * paranoia (Boolean) - Log out users if their IP address changed.
 * permit_crawling (Boolean) - Restricts the search for not logged in users to one search per minute.
+* profiler (Boolean) - Enable internal timings to help optimize code. Needed for "rendertime" addon. Default is false.
```

Comment:
Can you have a look if the *One line per sentence* rule breaks lists? If not it should be applied here too I think.